src/web/server/htmlTemplate.ts

```diff
@@ -154,6 +154,14 @@ export const htmlTemplate = ({
                     window.guardian.gaPath = "${gaPath.legacy}";
                 </script>
 
+                <script>
+                    // Noop monkey patch perf.mark and perf.measure if not supported
+                    if(window.performance.mark === undefined) {
```

Comment:
What happens here if `window.performance` is undefined?

Reply:
https://github.com/guardian/dotcom-rendering/pull/2269/commits/5adeb76410ca78179756ce68fa20f96de614d57c